table/tables/tables.go

```diff
@@ -760,9 +760,9 @@ func (t *TableCommon) RemoveRecord(ctx sessionctx.Context, h int64, r []types.Da
 		}
 		err = t.addDeleteBinlog(ctx, binlogRow, colIDs)
 	}
-	colSize := make(map[int64]int64, len(t.Cols()))
```

Comment:
Why changes this?

Reply:
If we use `Cols()` here, when exec `delete from t;` on the table with contains hidden columns.
server will panic.